pmlc/dialect/tile/tests/translate-to-stripe.mlir

```diff
@@ -13,9 +13,9 @@ func @eltwise_add(
 
 // CHECK: 0:   #program
 // CHECK-NEXT: block []:1 ( // eltwise_add
-// CHECK-NEXT:     #user none new@0x00000000 _X0[0, 0] fp32:I(10, 20):(20, 1):800 B
-// CHECK-NEXT:     #user none new@0x00000000 _X1[0, 0] fp32:I(10, 20):(20, 1):800 B
-// CHECK-NEXT:     #user none new@0x00000000 _X2[0, 0] fp32:I(10, 20):(20, 1):800 B
+// CHECK-NEXT:     none new@0x00000000 _X0[0, 0] fp32:I(10, 20):(20, 1):800 B
+// CHECK-NEXT:     none new@0x00000000 _X1[0, 0] fp32:I(10, 20):(20, 1):800 B
+// CHECK-NEXT:     none new@0x00000000 _X2[0, 0] fp32:I(10, 20):(20, 1):800 B
```

Comment:
This seems wrong, don't we want to require that the `#user` tag is set somehow?

Reply:
Yes, but that's not what the current Tile lowerer does; IMHO, MLIR transcoding should respect what's there, and not unconditionally set `#user`.  (In Stripe Classic, this is being done at `gen_stripe.cc:141` and `gen_stripe.cc:80`.)

Reply:
I'm saying can we change the test so that the input has something that causes the `#user` tag to be set. This will be a bug without more code...